presto-docs/src/main/sphinx/release/release-0.182.rst

```diff
@@ -0,0 +1,42 @@
+=============
+Release 0.182
+=============
+
+General Changes
+---------------
+
+* Add :doc:`/connector/tpcds`. This connector provides a set of schemas to
+  support the TPC Benchmark™ DS (TPC-DS).
+* Add a new :ref:`ipaddress_type` type to represent IP addresses.
+* Add a new experimental ORC writer implementation optimized for Presto.
+  We have some upcoming improvements, so we recommend waiting a few releases before
+  using this in production. The new writer can be enabled with the
+  ``hive.orc.optimized-writer.enabled`` configuration property or with the
+  ``orc_optimized_writer_enabled`` session property.
+* Add :func:`to_milliseconds` function to convert intervals (day to second) to milliseconds.
+* Add support for column aliases in ``CREATE TABLE ... AS`` statements.
```

Comment:
Write this as `CREATE TABLE AS` since that's what we call it in the SQL Statement Syntax section. Or make it a link:
```
:doc:`/sql/create-table-as`